microservices/case-service/src/main/java/org/entando/bundle/service/impl/ExternalServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package org.entando.bundle.service.impl;
+
+import org.entando.bundle.repository.CaseRepository;
+import org.entando.bundle.service.ExternalService;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+@Service
+public class ExternalServiceImpl implements ExternalService {
```

Comment:
Call this `CaseIdentityService` to make it more explicit

Reply:
Done